### PR TITLE
[TW-341] Updates for Environment Merging and PRs

### PR DESCRIPTION
**Pull request by claireriley**
* Adds brief sections to /docs/sending-requests/managing-environments/ (based on /docs/sending-requests/intro-to-collections/)
* Updates /docs/collaborating-in-postman/version-control-for-collections/ to account for environments having a similar fork/PR/merge experience
* Maintaining diff for review, but planning to update the page URL to remove "for collections"

It seems too wordy to say "collection or environment" everywhere, hence the frequent use of "entity"—thoughts?

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 1]
```diff
@@ -50,69 +49,48 @@ Postman's version control features provide a way for you and your team to collab
...
-When you fork a Postman collection, you create a copy of it in a different workspace.
+When you fork a Postman collection or environment, you create a copy of it in a different workspace. You must be signed in create a fork.
 
-To fork a collection:
+To fork a collection or environment:
 
-1. Select **Collections** in the left sidebar.
-1. Select the three dots <img alt="Three dots icon" src="https://assets.postman.com/postman-docs/icon-three-dots-v9.jpg" width="18px" style="vertical-align:middle;margin-bottom:5px"> next to the collection name.
+1. Hover over the entity in the left sidebar.
+1. Select the three dots <img alt="Three dots icon" src="https://assets.postman.com/postman-docs/icon-three-dots-v9.jpg" width="18px" style="vertical-align:middle;margin-bottom:5px"> that appear next to its name.
```
Comment (ankitjaggi) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:65]: Instead of the three dots option, can we point them to use the Fork Button that shows up when they open an element in a tab.

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 2]
```diff
@@ -199,16 +176,16 @@ There are two different types of checks that you can enable for a pull request:
 
 To set merge checks for pull requests on a specific collection:
 
-1. Select the collection in the **Collections** sidebar.
-1. Select the three dots <img alt="Three dots icon" src="https://assets.postman.com/postman-docs/icon-three-dots-v9.jpg" width="18px" style="vertical-align:middle;margin-bottom:5px"> next to the collection name.
+1. Hover over the entity in the left sidebar.
```
Comment (ankitjaggi) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:179]: You can keep this to collection as is. Merge checks is not yet enabled for environment PRs.

**Review by ankitjaggi — CHANGES_REQUESTED** [review]

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md [hunk 3]
```diff
@@ -50,69 +49,48 @@ Postman's version control features provide a way for you and your team to collab
     * [Merging changes](#merging-changes)
 * [Resolving conflicts](#resolving-conflicts)
 
-## Forking Postman elements
+## Forking Postman entities
 
-A _fork_ is a new instance of an element that you can modify without making any changes to the parent element. In Postman, you can fork [collections](#forking-a-collection) and [environments](#forking-an-environment). Forking also enables you to contribute to a collection or an environment without having Editor access to that element.
+A _fork_ is a new instance of an element that you can modify without making any changes to the parent element. In Postman, you can fork collections and environments. Forking also enables you to contribute to a collection or an environment without having [editor access](/docs/collaborating-in-postman/roles-and-permissions/#element-based-roles) to that element.
```
Comment (HeatherStenson) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:54]: Should "elements" --> "entities" here to match the heading change?

**Review by HeatherStenson — APPROVED** [review]
These changes look really good! 🍊

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md (hunk 3 above)
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/version-control-for-collections.md:54]: Change these 3 instances of "element" to "entity" to be consistent?

**Review by mark-anthony-postman — APPROVED** [review]
LGTM! 🌟  Just made one minor comment.

File: src/pages/docs/collaborating-in-postman/version-control-for-collections.md (hunk 3 above)
Comment (mark-anthony-postman) [reply src/pages/docs/collaborating-in-postman/version-control-for-collections.md:54]: D'oh, Heather beat me to it! 😄

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

**Review by ankitjaggi — APPROVED** [review]
